morpheus/_lib/src/python_modules/messages.cpp

```diff
@@ -219,10 +239,24 @@ PYBIND11_MODULE(messages, m)
         .def_property_readonly("count", &MultiInferenceFILMessageInterfaceProxy::count);
 
     py::class_<TensorMemory, std::shared_ptr<TensorMemory>>(m, "TensorMemory")
-        .def_readonly("count", &TensorMemory::count);
+        .def(py::init<>(&TensorMemoryInterfaceProxy::init),
+             py::arg("count"),
+             py::arg("tensors") = CupyUtil::py_tensor_map_t())
```

Comment:
Lets remove the `.tensors` property in favor of explicit `set_tensors()` and `get_tensors()` properties.